pom.xml

```diff
@@ -178,6 +203,16 @@
             <artifactId>jackson-datatype-trove</artifactId>
             <version>0.0.5</version>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+            <version>2.3.1</version>
+        </dependency>
```

Comment:
why are those in here? didn't Hermann have problems with the xml-bind library in Java 11?

Reply:
I added these because it turns out that these two libs are provided only in JDK 6/7/8.

They are still part of JDK 9 & 10 (marked as `@Deprecated(forRemoval=true)`) but are not in the classpath so we must use `--add-modules java.xml.bind` when running, to add them.

In JDK11 they are completely removed.

For more see: https://stackoverflow.com/a/43574427/10017209

Reply:
and what do we need them for (asking out of interest)? I don't see any trace of them in the projects

Reply:
they are used by jersey dependencies (probably some others too, but for jersey i'm sure)

Reply:
Finally removed the above libraries :).

Turned out that `jaxb-runtime` was not used by jersey but by hibernate. Increasing hibernate version fixed the problem (in core & stats). `NoClassDefFoundError: javax/xml/bind/JAXBException` happened while running java-client tests only happened locally at my machine.

 Clearing `/.m2/repository` and reimporting mvn dependencies solved the problem (don't know why but it works :P ).